ui/src/components/terminal/TerminalDialog.vue

```diff
@@ -133,7 +142,7 @@ export default {
         })
         .join('&');
 
-      this.ws = new WebSocket(`ws://${location.host}/ws/ssh?${params}`);
+      this.ws = new WebSocket(this.protocolConnectionURL+`://${location.host}/ws/ssh?${params}`);
```

Comment:
As this seems to be used only here, wouldn't be better to use a local variable here and use it to generate the URL?

Reply:
@leonardojoao

Reply:
I did it this way to avoid asynchronism, it was guaranteed that we would not have a problem.